cobra/core/model.py

```diff
@@ -1020,3 +1019,35 @@ def merge(self, right, prefix_existing=None, inplace=True,
             sum=self.objective.expression + right.objective.expression
         )[objective]
         return new_model
+
+    def _repr_html_(self):
+        return """
+        <table>
+            <tr>
+                <td><strong>Name</strong></td>
+                <td>{name}</td>
+            </tr><tr>
+                <td><strong>Memory address</strong></td>
+                <td>{address}</td>
+            </tr><tr>
+                <td><strong>Number of metabolites</strong></td>
+                <td>{num_metabolites}</td>
+            </tr><tr>
+                <td><strong>Number of reactions</strong></td>
+                <td>{num_reactions}</td>
+            </tr><tr>
+                <td><strong>Objective expression</strong></td>
+                <td>{objective}</td>
+            </tr><tr>
+                <td><strong>Compartments</strong></td>
+                <td>{compartments}</td>
+            </tr>
+          </table>""".format(
+            name=self.id,
+            address='0x0%x' % id(self),
+            num_metabolites=len(self.metabolites),
+            num_reactions=len(self.reactions),
+            objective=str(self.objective.expression),
+            compartments=", ".join(c for _, c in iteritems(self.compartments))
```

Comment:
Yes of course 💦  but just realised values might not be defined so better use combination